### ISSUE TITLE
`FileTrigger` does not apply aria attributes to input element

### ISSUE THREAD

**Issue by alizehkhan** [adobe/react-spectrum#7746]
### Provide a general summary of the issue here

Aria attributes do not apply to the input element for the `FileTrigger` component.

### 🤔 Expected Behavior?

```tsx
<FileTrigger aria-label="Upload File"></FileTrigger>
```
Input should render with aria-label like so:
```tsx
 <input
      aria-label="Upload File" // <----aria-label present in the dom
      class="react-aria-Input"
      data-rac=""
      style="display: none;"
      type="file"
    />
```

### 😯 Current Behavior

```tsx
<FileTrigger aria-label="Upload File"></FileTrigger>
```
Renders input without aria-label
```tsx
 <input
      // missing aria-label!
      class="react-aria-Input"
      data-rac=""
      style="display: none;"
      type="file"
    />
```

### 💁 Possible Solution

Add `{ labelable: true }` to `filterDOMProps` arguments for `FileTrigger` component 

### 🖥️ Steps to Reproduce

This test fails...
```tsx
  it("should accept aria attributes on the input", async () => {
    let { getByLabelText } = render(
      <FileTrigger aria-label="File Upload">
        <Button>Upload</Button>
      </FileTrigger>
    );

    let file = new File(["hello"], "hello.png", { type: "image/png" });

    await userEvent.upload(getByLabelText("File Upload"), file);
  });
```

### Version

1.6.0 react-aria-components

### What browsers are you seeing the problem on?

Chrome

### What operating system are you using?

MacOS

**Comment by snowystinger** [comment]
The input is `display: none`, so I'm not sure why it should have an `aria-label`, it's not in the accessibility tree and you can't navigate to it. Is this only for tests being able to do 'getByLabelText'? Or are you running into some other issue?

You should a button like we show in the docs https://react-spectrum.adobe.com/react-aria/FileTrigger.html#filetrigger

**Comment by alizehkhan** [comment]
Ahh true, it's not in the tree. Yeah, this was for my tests. I'll close this issue and the PR.
